plugins/home-page/package.json

```diff
@@ -8,27 +8,21 @@
   "scripts": {
     "build": "backstage-cli plugin:build",
     "lint": "backstage-cli lint",
-    "test": "backstage-cli test", 
+    "test": "backstage-cli test",
     "clean": "backstage-cli clean"
   },
   "devDependencies": {
     "@backstage/cli": "^0.1.1-alpha.4",
-    "@backstage/core": "^0.1.1-alpha.4",
-    "@backstage/theme": "^0.1.1-alpha.4",
-    "@material-ui/core": "^4.9.1",
-    "@material-ui/icons": "^4.9.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "@types/jest": "^24.0.0",
     "@types/node": "^12.0.0",
-    "@types/testing-library__jest-dom": "5.0.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0"
+    "@types/testing-library__jest-dom": "5.0.2"
   },
-  "peerDependencies": {
-    "@backstage/core": "^0.1.1-alpha.3",
-    "@backstage/theme": "^0.1.1-alpha.3",
+  "dependencies": {
+    "@backstage/core": "^0.1.1-alpha.4",
```

Comment:
Good catch on the version 👍, I wonder if lerna missed bumping them because there were duplicates? 🤔